bin/openstack_taster

```diff
@@ -101,6 +112,6 @@ image   = Fog::Image::OpenStack.new(OPENSTACK_CREDS)
 network = Fog::Network::OpenStack.new(OPENSTACK_CREDS)
 
 exit OpenStackTaster.new(
-  compute, volume, image, network, ENV['OS_NETWORK_REF'], flavor,
+  compute, volume, image, network, ENV['OS_NETWORK_REF'], settings[:flavor],
   SSH_KEYS, LOG_DIR
-).taste(image_name, settings)
+).taste(settings[:image_name], settings)
```

Comment:
It might make more sense to only pass the `settings` hash if we are only pulling variables from that, but this is something we can do later.